draft-ietf-rats-corim.md

```diff
@@ -162,19 +162,22 @@ and {{Section G of -cddl}}. Terms and concepts are always referenced as proper n
 
 # Verifier Reconciliation {#sec-verifier-rec}
 
-This specification describes the CoRIM format and documents how a Verifier should process the CoRIM to enable CoRIM authors to convey their intended meaning.
+This specification describes the CoRIM format and documents how a Verifier should process the CoRIM.
+This enables CoRIM authors to use CoRIM such that it conveys its intended meaning.
```

Comment:
Is this really making it better? That CoRIM enables CoRIM authors to convey their intended meaning is like saying water is wet. What does the reader glean from 166 that it not obvious anyways?

I.e., I like the change, it render removing 166 more a more obvious choice :-)

Reply:
> I.e., I like the change, it renders removing 166 more of an obvious choice 😄

Agree.  At the same time, I'd use this PR to add some meat to L165.

First, since the goal is the determinism of a CoRIM verifier, I suggest s/should process/must process/

Second, since the goal is the determinism of a CoRIM verifier, I suggest making the point explicit.